templates/base.html

```diff
@@ -102,36 +102,40 @@
 
         <!-- Success Modal -->
         <div class="reveal" id="success-modal" data-reveal>
-            <h1>Done</h1>
-            <p class="success-message">The action completed successfully.</p>
-            <button class="button" data-close>Okay, got it!</button>
+            <h1 class="white-color center">DONE</h1>
+            <p class="success-message white-color center">The action completed successfully.</p>
+            <div class="center">
+                <button class="button center" data-close>Okay, got it!</button>
+            </div>
             <button class="close-button" data-close aria-label="Close modal" type="button">
                 <span aria-hidden="true">&times;</span>
             </button>
         </div>
 
         <!-- Error Modal -->
         <div class="reveal" id="error-modal" data-reveal>
-            <h1>Error</h1>
+            <h1 class="white-color center">Error</h1>
```

Comment:
If you're going to put `DONE` above in all caps, might as well make `ERROR` all caps for consistency.